Source/Core/Common/Timer.cpp

```diff
@@ -261,4 +261,13 @@ double Timer::GetDoubleTime()
 	return TmpTime;
 }
 
+std::string Timer::GetDateTimeFormatted(double time)
+{
+	time_t seconds = (time_t)time + (38 * 365 * 24 * 60 * 60);
```

Comment:
Unless you can get rid of that, maybe make that multiplication result a constant? Just to make it easier to see why we add 38 years to this.